.env.docker

```diff
@@ -80,3 +80,15 @@ BLACKFIRE_SERVER_TOKEN=<your_backfire_secret>
 
 WEBHOOK_BASIC_AUTH_USERNAME=webhook
 WEBHOOK_BASIC_AUTH_PASSWORD=cCVMxdX8hfctDefk
+
+RATE_LIMITER_STORAGE_DSN=redis://simple-roster-redis/1
+RATE_LIMITER_FIXED_WINDOW_ROUTES="updateLineItems"
+RATE_LIMITER_FIXED_WINDOW_INTERVAL="2 minutes"
+RATE_LIMITER_FIXED_WINDOW_LIMIT=3
```

Comment:
I am wondering if need more flexibility here. For now I think it's ok to have one global setting, but later we might need to have a configuration file in place where it is possible to define limits per routes.

Also we'll have to be mindful about the default values on prod, we'll need something like 10 requests / second, to be discussed with the team.

Reply:
I fully agree. If during the usage we see that this configuration is not enough, we can improve it for sure. 
Definitely, we have to discuss these parameters when we go to the production environment with the team :)